src/main/java/me/botsko/prism/actions/BlockAction.java

```diff
@@ -30,574 +30,534 @@
 import org.bukkit.entity.EntityType;
 import org.bukkit.entity.Player;
 
+import static org.bukkit.Material.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using the '.*' form of import should be avoided - org.bukkit.Material.*.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5446892)